openraft/tests/append_entries/t50_append_entries_with_bigger_term.rs

```diff
@@ -0,0 +1,64 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use maplit::btreeset;
+use openraft::raft::AppendEntriesRequest;
+use openraft::Config;
+use openraft::LogId;
+use openraft::RaftNetwork;
+use openraft::State;
+
+use crate::fixtures::RaftRouter;
+
+/// append-entries should update hard state when adding new logs with bigger term
+///
+/// - bring up a learner and send to it append_entries request. Check the hard state updated.
+#[tokio::test(flavor = "multi_thread", worker_threads = 6)]
+async fn append_entries_with_bigger_term() -> Result<()> {
+  let (_log_guard, ut_span) = init_ut!();
+  let _ent = ut_span.enter();
+
+  // Setup test dependencies.
```

Comment:
`make fmt` will format everybody's code in a universal way. :D

And it's better to turn on `format on save` in your IDE and make it use `rustfmt` instead of its internal formatter.

Clion has these options:

![image](https://user-images.githubusercontent.com/44069/149067664-b10e45d1-eb58-43d6-93d4-54e8cbe9932c.png)